news/editions/edition3.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: single
+title:  "*Flying*  Edition 3"
+sidebar:
+  nav: news
+redirect_from: /news/latest/
+---
+## Open Source: #BetterTogether
+> This month's issue focuses Spinnaker's strong open source DNA. The OSS community embraces Spinnaker, as it builds a bridge between the collaboration and interoperability story at the core of Linux culture, and the powerful cloud economy that has leveraged it to transform software delivery. Efforts to build native Kubernetes integration highlight this unique strength.
+
+## [The New Age of the K8S Operator](https://www.armory.io/blog/introducing-spinnaker-operator-a-kubernetes-native-blueprint-for-success/)
+Spinnaker Operator, a Kubernetes operator for Spinnaker, has reached Beta. With it installed, you can use kubectl commands to install, deploy, and upgrade any version of Spinnaker through its lifecycle. Read the [doc](https://github.com/armory/spinnaker-operator/blob/release-0.3.x/README.md), try it out, and use the #kubernetes-operator channel in Spinnaker Slack to share your feedback.
```

Comment:
> through its lifecycle.

This is confusing. Spinnaker's lifecycle? kubectl's? The Operator's?

Reply:
Is "...through its lifecycle" necessary?